tests/responses/subscriptions_for_userid.json

```diff
@@ -0,0 +1,21 @@
+{
+  "count": 1,
+  "previous": null,
+  "results": [{
+    "status": "active",
+    "deprovision_date": "2016-11-17",
+    "term": "P1Y",
+    "user_id": "fake-user-id-abcdef1234567890abc",
+    "auto_renew": false,
+    "billing_date": "2016-11-02",
+    "properties": {
+      "partner_id": "WL_BOTTOMLINE"
+    },
+    "expiry_date": "2016-11-02",
+    "start_date": "2015-11-02",
+    "sku": "12345678",
+    "type": "wl_subscription_type",
+    "id": "fake-sub-id-abcdef1234567890abcd"
+  }],
+  "next": null
+}
```

Comment:
Opinion time: The keys here matter for the model's init and the value of the id is being asserted in the test. So I would prefer if this data lived in the the test case. I don't see any benefit to hiding it away other than aesthetics. Aesthetics that, in my opinion, take away from the value of the tests. What do you think?